stan/math/opencl/prim/normal_id_glm_lpdf.hpp

```diff
@@ -95,14 +95,13 @@ return_type_t<T_alpha, T_beta, T_scale> normal_id_glm_lpdf(
   const auto& beta_ref = to_ref_if<!is_constant<T_beta>::value>(beta);
   const auto& alpha_ref = to_ref_if<!is_constant<T_alpha>::value>(alpha);
 
-  const auto &beta_val = value_of_rec(beta_ref);
-  const auto &alpha_val = value_of_rec(alpha_ref);
-  const auto &sigma_val = value_of_rec(sigma_ref);
+  const auto& beta_val = value_of_rec(beta_ref);
+  const auto& alpha_val = value_of_rec(alpha_ref);
+  const auto& sigma_val = value_of_rec(sigma_ref);
 
   const auto& beta_val_vec = as_column_vector_or_scalar(beta_val);
   const auto& alpha_val_vec = as_column_vector_or_scalar(alpha_val);
-  const auto& sigma_val_vec
-      = to_ref(as_column_vector_or_scalar(sigma_val));
+  const auto& sigma_val_vec = to_ref(as_column_vector_or_scalar(sigma_val));
```

Comment:
I don't see anything that's changed other than formatting in this PR.   What in this PR changes the behavior?

Also, if there are new signatures accepted, they should at least be tested for instantiation in the unit tests.

Reply:
While I was doing the changes I also fixed the formatting in the few cases that were prevoiusly missed. I see that newer clang-format fixed some more. Most important changes include the fact that the arguments are evaluated if they are expressions and the way `operands_and_partials` object is constructed.

Adding tests for every function independantly is a lot of work. So I opted not to do it. Instead I and @rok-cesnovar are working on a semi-automatically generated test that will do that check all signatures exposed to stan language. Once all functions are generalized we will add it into Jenkins pipeline.

Reply:
To clarify, you mean you opted not to do tests if every function works with expressions.
This does not add any new GLM signatures?

Reply:
Exactly.

Reply:
Is this PR just formatting?  If not, can you point me to what changed?  I lost the game of "Where's Waldo" to find the change myself.

> Adding tests for every function independantly is a lot of work. So I opted not to do it. 

I'm not sure what this means.  It sounds like you're proposing to add new functions without tests in the hope that the tests will get added later.  I'd rather do that the other way around and add the test framework before the new functions.

Reply:
> Is this PR just formatting?
80% of changed lines are functional changes. If you want to ignore changes to formatting, github has an option to hide whitespace changes - click on that wheel at the top when looking at changes. 

>  It sounds like you're proposing to add new functions without tests in the hope that the tests will get added later.

I am making existing functions also accept Eigen expressions. I admit it is still a similar situation. However, nothing will rely on these changes untill tests are in. 

>  I'd rather do that the other way around and add the test framework before the new functions.

If I added the tests now they would fail. If I cramed all generalization changes in a single PR together with tests it would touch half of Stan Math at once - to much to review at one time.